streaming/src/main/scala/io/finch/streaming/package.scala

```diff
@@ -0,0 +1,49 @@
+package io.finch
+
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.Endpoint.Result
+import io.finch.items.RequestItem
+import io.iteratee.{Enumeratee, Enumerator}
+
+/**
+  * Streaming module
+  */
+package object streaming {

```

Comment:
We can just point monad instance to one deinfed din catbird. I'm ok not doing this now (extending module), but I think it's a right thing to do at the end of the day.